src/main/java/com/iridium/iridiumskyblock/managers/IslandManager.java

```diff
@@ -233,6 +234,24 @@ public boolean getIslandPermission(@NotNull Island island, @NotNull User user, @
         return getIslandPermission(island, islandRank, permission) || user.isBypass();
     }
 
+    /**
+     * Gets an Island's bank from BankItem
+     *
+     * @param island   The specified Island
+     * @param bankItem The BankItem we are getting
+     * @return the IslandBank
+     */
+    public IslandBank getIslandBank(@NotNull Island island, @NotNull BankItem bankItem) {
+        Optional<IslandBank> optionalIslandBank = IridiumSkyblock.getInstance().getDatabaseManager().getIslandBankList().stream().filter(isBank -> isBank.getIsland() == island.getId() && isBank.getBankItem().equalsIgnoreCase(bankItem.getName())).findFirst();
```

Comment:
isBank sounds like an accessor for a boolean, islandBank or just bank would be better choices